src/System.Runtime/tests/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace System.Tests
+{
+    public static class FormatterServicesTests
+    {
+    	private static Func<Type, Object> geo = FormatterServices.GetUninitializedObject;
+
+        [Fact]
+        public static void NoArgument()
+        {
+            Assert.Throws<ArgumentNullException>(() => geo(null));
```

Comment:
Nit: With Argument*Exceptions, Assert.Throws has overloads where the first parameter is the name of the parameter with which the exception is associated.  It's good to pass that in as well so xunit validates it.